docs/usage/configuration.rst

```diff
@@ -165,6 +165,19 @@ Default::
 
     'aol'
 
+.. setting:: TEXTFSM_TEMPLATE_DIR
+
+TEXTFSM_TEMPLATE_DIR
+~~~~~~~~~~~~~
+
+Default path to TextFSM template directory. It is recommended to pull the Network to Code templates
+from here ``https://github.com/networktocode/ntc-ansible/tree/master/ntc_templates`` and place them inside
```

Comment:
Ah no probs! Done and Done.  
  

  

Regards,

  

Thomas Cuthbert

> On Feb 26 2016, at 1:16 am, Jathan McCollum &lt;notifications@github.com&gt;
wrote:  

>

> In [docs/usage/configuration.rst](https://github.com/trigger/trigger/pull/24
9#discussion_r54129754):

>  
>  
>     > @@ -165,6 +165,19 @@ Default::

>     >  
>     >      'aol'

>     >  
>     > +.. setting:: TEXTFSM_TEMPLATE_DIR

>     > +

>     > +TEXTFSM_TEMPLATE_DIR

>     > +~~~~~~~~~~~~~

>     > +

>     > +Default path to TextFSM template directory. It is recommended to pull
the Network to Code templates

>     > +from here ``https://github.com/networktocode/ntc-
ansible/tree/master/ntc_templates`` and place them inside

>  
>

> Let's make this a link instead of a code.

>

> Change this:

>  
>  
>     Network to Code templates from here ``https://github.com/networktocode
/ntc-ansible/tree/master/ntc_templates``

>  
>

> To this:

>  
>  
>     `Network to Code templates <https://github.com/networktocode/ntc-
ansible/tree/master/ntc_templates>`_

>  
>     Which will render it as a hyperlink. :)

>  
>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/trigger
/trigger/pull/249/files#r54129754).![](https://github.com/notifications/beacon
/AD64L98zwuXpaSXTxELsviZIHVpAvXNBks5pny5QgaJpZM4HeiI4.gif)